assets/constants.js

```diff
@@ -527,7 +527,7 @@ var constants = {
 
   "ENABLE_PREREQUISITE_SKILLS": false,
 
-  "ENABLE_NEW_STRUCTURE_PLAYERS": false,
+  "ENABLE_NEW_STRUCTURE_PLAYERS": true,
```

Comment:
Why do this? I think this is completely unrelated...

Reply:
Yeah, actually for some purpose I do this. And by mistake, its comes under commit.
I will fix it.